src/main/java/teammates/common/datatransfer/CommentAttributes.java

```diff
@@ -48,7 +48,7 @@ public CommentAttributes(String courseId, String giverEmail, CommentParticipantT
         this.giverEmail = giverEmail;
         this.recipientType = recipientType == null ? CommentParticipantType.PERSON : recipientType;
         this.recipients = recipients;
-        this.commentText = commentText;
+        this.commentText = commentText == null ? null : new Text(Sanitizer.sanitizeForRichText(commentText.getValue()));
```

Comment:
What about here (also repeated in several other places)? Can we abstract this repetitive use of ternary operator to a utility method as well?